assets/css/sc-welcome-pg-style.css

```diff
@@ -0,0 +1,69 @@
+#wpwrap {
+	background-color: #f2f7fb;
+}
+
+#simcal-welcome .wp-person a:focus .gravatar,
+#simcal-welcome a:focus,
+#simcal-welcome a:focus .media-icon img {
+	box-shadow: unset;
+}
+
+#simcal-welcome a.button.button-primary {
+	background: rgba(255, 255, 255, 0.2);
+	border-radius: 100px;
+	border: 1px solid white;
+	margin: 6px;
+}
+
+#simcal-welcome .about-text {
+	margin: 15px 0px 0px 0px;
+	color: white;
+}
+
+#simcal-welcome .whats-new-wrap img {
+	border: unset;
+	margin: unset;
+}
+
+#simcal-welcome .about-wrap img {
+	width: unset;
+}
+
+#simcal-welcome .about-wrap .wp-person .gravatar {
+	border-radius: 50%;
+	height: 50px;
+	padding: 0px;
+}
+
+#simcal-welcome .about-wrap .wp-person .web {
+	font-weight: 500;
+	font-size: 14px;
+	color: #000000;
+}
+
+#simcal-welcome .about-wrap .wp-person {
+	width: 23%;
+}
+
+#simcal-welcome a.button.button-primary:hover {
+	background-color: white;
+	color: #60bc4e;
+}
+
+#simcal-welcome .nav-tab-active {
+	color: #3a414c;
+}
+
+#simcal-welcome .wp-person:hover .web {
+	color: #fff;
+}
+
+#simcal-welcome .wp-person:hover img {
+	border: 2px solid white;
+}
+
+@media only screen and (max-width: 1765px) {
+	#simcal-welcome .about-wrap .wp-person {
+		width: 28%;
```

Comment:
This can be done using tailwind classes as well

Reply:
we are writing any line after this .wp-person , its giving critical error.

Reply:
I did not understood

Reply:
it is not taking tailwind css first , first css applied .wp-person class and than tailwind css applied.

Reply:
we can solve through appling the css important to that css in tailwind because it is wp-person generated by the wordpress.